src/compas_fab/robots/planning_scene.py

```diff
@@ -92,10 +120,17 @@ def __init__(self, collision_mesh, link_name, touch_links=None, weight=1.):
 class PlanningScene(object):
     """Represents the planning scene.
 
+    Parameters
+    ----------
+    robot : :class:`Robot`
+        A reference to the robot in the planning scene.
+
     Attributes
     ----------
     robot : :class:`Robot`
         A reference to the robot in the planning scene.
+    client : :class:`compas_fab.backend.RosClient` or :class:`compas_fab.backend.VrepClient`
```

Comment:
Don't refer to specific implementations here. It doesn't add much value and it would mean there's more effort in the event of adding a new client.

Reply:
From a documentation standpoint having a parent class for backend clients would make sense here. Seems like that might be part of #149. Until that's implemented we can just skip type hint.

Reply:
yep